### ISSUE TITLE
Improve blog post url

### ISSUE THREAD

**Issue by fdrouet** [exo-addons/blog-extension#16]
find a way to improve the blog post url to have something without parameters :

```
http://int.exoplatform.org/portal/intranet/blog/article?content-id=/repository/collaboration/Groups/platform/users/Documents/2012/11/23/my-blog-post
```

should be for exemple

```
http://int.exoplatform.org/portal/intranet/blog/article/2012/11/23/my-blog-post
```

**Comment by plamarque** [comment]
I have a plan to use the friendly url service for that. It will not produce exactly the URL that you mentioned, but we could have something like : http://int.exoplatform.org/portal/content/blog/article/2012/11/23/my-blog-post

**Comment by fdrouet** [comment]
the Navigation Controller should be the good way also

**Comment by plamarque** [comment]
if only it was possible to extend via en extension.... which is not the case afaik....

**Comment by fdrouet** [comment]
yes I know but it could be a manual addition in the controller.xml file during the install phase

**Comment by plamarque** [comment]
I think I'll go with the friendly url service approach and see what it gives. it looks easier to do and will work out of the box.

Feel free to send a PR for navigation controller integration though!
